src/input.rs

```diff
@@ -105,17 +105,21 @@ pub struct Opts {
     #[structopt(long, possible_values = &ScanOrder::variants(), case_insensitive = true, default_value = "serial")]
     pub scan_order: ScanOrder,
 
+    /// Use the top 1000 ports.
+    #[structopt(long)]
+    pub top: bool,
```

Comment:
Is this part of this PR?

Reply:
This was part of an earlier PR, I just changed up the location of it.